go.mod

```diff
@@ -267,4 +267,4 @@ require (
 	stash.kopano.io/kgol/kcc-go/v5 v5.0.1 // indirect
 )
 
-replace github.com/cs3org/go-cs3apis => github.com/micbar/go-cs3apis v0.0.0-20220617090231-703c04619761 // temp fork
+replace github.com/cs3org/reva/v2 => github.com/micbar/reva/v2 v2.0.0-20220712083851-2832f4d5031e
```

Comment:
Can this be replaced with current edge now that corresponding reva PR is merged?

Reply:
should be, yes